Shared/API/CloudFunction.swift

```diff
@@ -11,11 +11,66 @@ import Parse
 import Combine
 
 protocol CloudFunction: StatusableRequest {
-    func makeRequest(andUpdate statusables: [Statusable], viewsToIgnore: [UIView]) -> AnyPublisher<ReturnType, Error>
+    func makeRequest(andUpdate statusables: [Statusable],
+                     viewsToIgnore: [UIView]) -> AnyPublisher<ReturnType, Error>
+    func makeAsyncRequest(andUpdate statusables: [Statusable],
+                          viewsToIgnore: [UIView]) async throws -> ReturnType
 }
 
 extension CloudFunction {
 
+    func makeAsyncRequest(andUpdate statusables: [Statusable],
```

Comment:
Following the pattern and creating an extension to handle boilerplate for updating statusables.